src/main/java/uk/ac/ebi/eva/metrics/util/MetricUtil.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ebi.eva.metrics.util;
+
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+public class MetricUtil {
```

Comment:
I suggest not having util classes as we might end up with a class with many different methods with different concerns. I would prefer to name it indicating the purpose of this, like calculate/generate/identifiers identifiers.

Also the classes that were specific to the eva-accessioning were moved to that repo. I think this one also classifies to be moved? In my opinion we either have the metric implementations for clustering and accessinioning here or move this to the eva-accession. In this case I would say to move this one

Reply:
moved the logic to specific metric-compute classes